client/client.go

```diff
@@ -5,6 +5,7 @@ import (
 	"errors"
 
 	"github.com/Azure/azure-sdk-for-go/services/subscription/mgmt/2020-09-01/subscription"
+	// Import all autorest modules
```

Comment:
Blank imports require a comments explaining them.
Depending if we do include a meaningful comment, we can keep the rule or remove it.